linodecli/plugins/k8s-alpha.py

```diff
@@ -75,6 +79,13 @@ def create(args, context):
         if 'kubectl' in needed_deps:
             print_kubectl_install_help()
         sys.exit(1)
+    # Check if terraform version is between min and max
+    if not terraform_version_supported(terraform_min_version,terraform_max_version):
+        print('Terraform version unsupported. Must be between v{} and v{}').format(
+                terraform_min_version,terraform_max_version)
```

Comment:
The close paren for the print statement should be after the close paren for `format` here: 

```python
        print('Terraform version unsupported. Must be between v{} and v{}'.format(
                terraform_min_version,terraform_max_version))

```